packages/babel-plugin-import-jsx-pragma/README.md

```diff
@@ -0,0 +1,90 @@
+Babel Plugin Import JSX Pragma
+======
+
+Babel transform plugin for automatically injecting an import to be used as the
+pragma for the [React JSX Transform plugin](http://babeljs.io/docs/en/babel-plugin-transform-react-jsx).
+
+[JSX](https://reactjs.org/docs/jsx-in-depth.html) is merely a syntactic sugar
+for a function call, typically to `React.createElement` when used with [React](https://reactjs.org/).
+As such, it requires that the function referenced by this transform be within
+the scope of the file where the JSX occurs. In a typical React project, this
+means React must be imported in any file where JSX exists.
+
+**Babel Plugin Import JSX Pragma** automates this process by introducing the
+necessary import automatically wherever JSX exists, allowing you to use JSX in
+your code without thinking to ensure the transformed function is within scope.
+
+## Installation
+
+Install the module to your project using [npm](https://www.npmjs.com/).
+
+```bash
+npm install @wordpress/babel-plugin-import-jsx-pragma
+```
+
+## Usage
+
+Refer to the [Babel Plugins documentation](http://babeljs.io/docs/en/plugins)
+if you don't yet have experience working with Babel plugins.
+
+Include `@wordpress/babel-plugin-import-jsx-pragma` as a plugin in your Babel
+configuration.
+
+It's assumed that you're also using [@babel/transform-react-jsx](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx/).
+Otherwise, you may encounter errors when encountering JSX tokens.
+
+```
```

Comment:
You should mark this as `js` so it'll syntax-highlight nicely on GitHub, eg:

<pre>
```js
// .babelrc.js
module.exports = {
	plugins: [
		'@wordpress/babel-plugin-import-jsx-pragma',
		'@babel/transform-react-jsx',
	],
};
```
</pre>